plugin/pkg/auth/authorizer/rbac/bootstrappolicy/controller_policy.go

```diff
@@ -221,8 +221,9 @@ func buildControllerRoles() ([]rbacv1.ClusterRole, []rbacv1.ClusterRoleBinding)
 	addControllerRole(&controllerRoles, &controllerRoleBindings, rbacv1.ClusterRole{
 		ObjectMeta: metav1.ObjectMeta{Name: saRolePrefix + "mizar-service-controller"},
 		Rules: []rbacv1.PolicyRule{
-			rbacv1helpers.NewRule("*").Groups("*").Resources("*").RuleOrDie(),
-			eventsRule(),
+			rbacv1helpers.NewRule("get", "list", "update", "watch").Groups(legacyGroup).Resources("services").RuleOrDie(),
+			rbacv1helpers.NewRule("get").Groups(legacyGroup).Resources("endpoints").RuleOrDie(),
+			rbacv1helpers.NewRule("get").Groups("arktos.futurewei.com").Resources("networks").RuleOrDie(),
```

Comment:
is this group name allowed to use arktos.futurewei.com ?

Reply:
Arktos network object belongs to group arktos.futurewei.com. It is existing objects